src/ripple/overlay/tests/manifest_test.cpp

```diff
@@ -122,6 +136,112 @@ class manifest_test : public ripple::TestSuite
         return Manifest (m.serialized, m.masterKey, m.signingKey, m.sequence);
     }
 
+
+    void
+    testConfigLoad ()
+    {
+        testcase ("Config Load");
+
+        ManifestCache cache;
+        beast::Journal journal;
+
+        std::vector<PublicKey> network;
+
+        while (network.size () != 8)
+            network.push_back (randomMasterKey());
+
+        auto format = [](
+            PublicKey const &publicKey,
+            char const* comment = nullptr)
+        {
+            auto ret = toBase58(
+                TokenType::TOKEN_NODE_PUBLIC,
+                publicKey);
+
+            if (comment)
+                ret += comment;
+
+            return ret;
+        };
+
+        Section s1;
+
+        // Correct (empty) configuration
+        expect (cache.loadValidatorKeys (s1, journal));
+        expect (cache.size() == 0);
+
+        // Correct configuration
+        s1.append (format (network[0]));
+        s1.append (format (network[1]) + " Comment");
```

Comment:
`format` takes a comment as an optional parameter. I'd use that instead of appending here.